package.json

```diff
@@ -18,7 +18,7 @@
   "license": "MIT",
   "dependencies": {
     "socket.io":"1.1.0",
-    "ember-inspector": "^1.6.3"
+    "ember-inspector": "emberjs/ember-inspector#master"
```

Comment:
I SUCK. Sorry :-( I've been struggling to find time to work on ember-related stuff the past months :-(